applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -393,6 +394,15 @@ int bt_mgmt_init(void)
 
 #endif /* CONFIG_AUDIO_BT_MGMT_DFU */
 
+#ifdef CONFIG_MCUMGR_TRANSPORT_BT_DYNAMIC_SVC_REGISTRATION
+	/* Unregister SMP server if DFU is not enabled */
```

Comment:
please write out SMP (Simple Management Protocol) to avoid confusion with Bluetooth.